source/index.txt

```diff
@@ -24,15 +24,11 @@ MongoDB Node Driver
 Introduction
 ------------
 
-Welcome to the official MongoDB Node.js driver documentation.
+Welcome to the documentation site for the official MongoDB Node.js 
+driver. You can add the driver to your application to work with MongoDB 
+in JavaScript. Download it using `npm <https://www.npmjs.com/>`__ 
```

Comment:
Alternatively, this could say:

You can add the driver to your Node.js application to work with MongoDB.